content/accessibility/index.md

```diff
@@ -17,7 +17,7 @@ Accessibility works best when incorporated into an agile production environment.
 
 The Web Content Accessibility Guidelines (WCAG) have long been the gold standard for accessibility on the web. Here at 18F we follow [WCAG2.0 AA](https://www.w3.org/TR/WCAG20/) as our minimum standard for accessibility.  
 
-If you have questions about accessibility or would like help, please reach out on the [GitHub Repo](https://github.com/18F/accessibility/TODO/).
+If you have questions about accessibility or would like help, please reach out on the [GitHub Repo](https://github.com/18F/guides/issues).
```

Comment:
@echappen Do you think we should be linking straight to an issue template here instead of the list of issues?

Reply:
This seems fine for now, thanks 👍